<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>062-940-3520</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>진행과정</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>기관위치</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>관리자 모드</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_class">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1건</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>대한상공회의소광주인력개발원</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>광주 광산구</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_training">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>09:00:00~17:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>통합심사과정훈련</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_menu">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>전체 공지</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>학생 일정</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>메신저</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>전화번호</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>훈련기관</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_class_name">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>데이터 기반 AI SW개발자 양성</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>과정명</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>금일훈련</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>훈련유형</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_page1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>751</width>
       <height>461</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>270</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>출석 확인</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>전체 공지</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_page2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>761</width>
       <height>361</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>공지 내역</string>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>510</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>현재 공지 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_notice">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>510</y>
       <width>671</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>문자 *</string>
     </property>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_input_page2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>580</y>
       <width>751</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_input_page2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>710</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>공지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_delet_page2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>710</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>내리기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_home_page2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_56">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>학생 일정</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTabWidget" name="tab_page3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>760</width>
       <height>520</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>달력</string>
      </attribute>
      <widget class="QCalendarWidget" name="calendar_page3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>755</width>
         <height>495</height>
        </rect>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>리스트 보기</string>
      </attribute>
      <widget class="QTableWidget" name="table_page3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>755</width>
         <height>495</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QLineEdit" name="le_input_page3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>660</y>
       <width>760</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_input_page3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>730</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_delet_page3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>730</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_home_page3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_name">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>이름 :</string>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>620</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_59">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>메신저</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_page4">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>60</y>
       <width>601</width>
       <height>590</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_input_page4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>680</y>
       <width>761</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_delet_page4">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>760</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_home_page4">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>20</width>
       <height>611</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>650</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_page4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>591</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
